### DISCUSSION TITLE
GitHub insights referral source as URL argument

### DISCUSSION THREAD

**Discussion by Fruchtzwerg94** [community/community#52080] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I like the way of getting traffic insights for my repository, where I can also see a list of referring sites.
![grafik](https://user-images.githubusercontent.com/29866610/230134535-e9634e8f-49d4-448c-a4c7-7703766159c0.png)

My plan is to set up my application, which may opens the repos site, as a referring source.

Therefore I am trying to understand how GitHub tracks the referral source. I assume [HTTP-Referer-Header](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Referer) is used. But since the closest documentation I can find ([API docs](https://docs.github.com/en/rest/metrics/traffic?apiVersion=2022-11-28)) isn't very specific about it its justs an assumption.

To allow integration in my application, I would like to include the referral source into the URL. Is there a way to achive this with additional URL arguments appended to the original repos URL? Maybe something like Googles `?utm_source=source_name` or more common `?ref=source_name` / `?referral=source_name`?

Thanks for your help.

**Comment by RobinRMC** [comment]
I would like to have this ability/feature as well.
